pkg/controllers/webhook/controller.go

```diff
@@ -766,6 +766,11 @@ func (c *controller) buildResourceValidatingWebhookConfiguration(ctx context.Con
 			return nil, err
 		}
 		c.recordPolicyState(config.ValidatingWebhookConfigurationName, policies...)
+		operationStatusMap := make(map[string]bool)
```

Comment:
faster to use array of bools indexed by enum.

Reply:
Yes Parikshit! Thank you!

Reply:
A map will be involved as far as I see. To use string constants as keys to an array, we can create a mapping between the string constants and their corresponding integer indices and then use it. Would you suggest that to be better?

Reply:
This part is ok Anushka.